cmd/utils/sub_command.go

```diff
@@ -8,55 +8,65 @@ import (
 
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/taikoxyz/taiko-client/cmd/logger"
+	"github.com/taikoxyz/taiko-client/driver"
 	"github.com/taikoxyz/taiko-client/metrics"
+	"github.com/taikoxyz/taiko-client/node"
+	"github.com/taikoxyz/taiko-client/proposer"
+	"github.com/taikoxyz/taiko-client/prover"
 	"github.com/urfave/cli/v2"
 )
 
-type SubcommandApplication interface {
-	InitFromCli(context.Context, *cli.Context) error
-	Name() string
-	Start() error
-	Close(context.Context)
+func StartServer(c *cli.Context) error {
```

Comment:
im not sure this is any simpler than the original method, and none of these applications are `Servers`, they are all different utility CLI apps?

Reply:
emmm dont see any simpler than the original method neither